Source/ACE.Server/Entity/Landblock.cs

```diff
@@ -155,28 +154,33 @@ private void CreateWorldObjects()
             var shardObjects = DatabaseManager.Shard.GetStaticObjectsByLandblock(Id.Landblock);
             var factoryObjects = WorldObjectFactory.CreateNewWorldObjects(objects, shardObjects);
 
-            // for mansion linking
-            var houses = new List<House>();
-
-            foreach (var fo in factoryObjects)
+            actionQueue.EnqueueAction(new ActionEventDelegate(() =>
```

Comment:
This breaks RootHouse, and anything that depends on the landblock synchronous loading functionality

With this line, CreateWorldObjects() returns immediately even when sync=true, effectively breaking anything that depends on the the objects to be loaded in before returning

Reply:
Is this resolved by the order patch?